Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -85,7 +85,9 @@ @implementation RCTLocationRequest
 
 - (void)dealloc
 {
-  [_timeoutTimer invalidate];
+  if([_timeoutTimer isValid]){
```

Comment:
Slightly prefer `_timeoutTimer.valid` since it's documented that way here: https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSTimer_Class/index.html#//apple_ref/occ/instp/NSTimer/valid